### PR TITLE
adding limit to InsertInvitationCodeViewController textfield

### PR DESCRIPTION
**Pull request by Perklone**

### CODE REVIEW

File: Sounds Like A Plan/Helpers/Helper.swift [hunk 1]
```diff
@@ -28,8 +28,8 @@ class Helper {
         
         let isOnBoarded: Bool = uDefaults.bool(forKey: "isOnboarded")
        
-        let vc = isOnBoarded ? tabBar: OnboardingContainerViewController()
-//        let vc = tabBar
+//        let vc = isOnBoarded ? tabBar: OnboardingContainerViewController()
```
Comment (gremlinflat) [inline Sounds Like A Plan/Helpers/Helper.swift:31]: revert this

File: Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift [hunk 2]
```diff
@@ -88,13 +95,37 @@ class InsertInvitationCodeViewController: UIViewController {
...
             joinButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -12)
         ])
-        
+        codeTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
         joinButton.addTarget(self, action: #selector(joinButtonPressed(_:)), for: .touchUpInside)
     }
+    
     @objc func joinButtonPressed(_ sender: UIButton){
-        if codeTextField.text?.count != 6 { return }
         delegate.searchEventFromCode(for: codeTextField.text ?? "626270")
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift:103]: better do guard let codeTextField.text instead nil coalesing

File: Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift [hunk 3]
```diff
@@ -19,6 +20,10 @@ class InsertInvitationCodeViewController: UIViewController {
     
     var delegate: InsertInvitationCodeVCDelegate!
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift:21]: strong reference, retain cycle possibly occurred during memory trottling
sol: do weak var delegate: xxxx

**Review by gremlinflat — CHANGES_REQUESTED** [review]
minor changes

**Comment by Perklone** [comment]
fixed

File: Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift (hunk 3 above)
Comment (gremlinflat) [reply Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift:21]: @Perklone

**Comment by gremlinflat** [comment]
retain cycle

File: Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift (hunk 3 above)
Comment (Perklone) [reply Sounds Like A Plan/Features/Join Event/InsertInvitationCodeViewController.swift:21]: force castingnya?

**Review by gremlinflat — APPROVED** [review]
LGTM 👍🏿

**Pull request merged**